src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/bulkscan/validation/out/OcrValidationResponse.java

```diff
@@ -1,25 +1,27 @@
-package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan;
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.validation.out;
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
 
 import java.util.List;
 
+@Getter
 public class OcrValidationResponse {
```

Comment:
->lib

Reply:
Moved into the BSP Commons Lib as requested